ducktape/command_line/main.py

```diff
@@ -60,15 +60,23 @@ def main():
     # in which all test results will be stored
     session_id = generate_session_id(ConsoleConfig.SESSION_ID_FILE)
     results_dir = generate_results_dir(session_id)
-    test_session_context = TestSessionContext(session_id, results_dir)
+    cluster = VagrantCluster()
+
+    # Make directory in which results will be stored
     if os.path.isdir(results_dir):
         raise Exception(
             "A test results directory with session id %s already exists. Exiting without overwriting..." % session_id)
     os.mkdir(results_dir)
+    latest_test_dir = "latest-test-results"
```

Comment:
This is definitely a bike-shedding comment, so feel free to ignore, but do we have to append "-test-results" to everything? If we want to label all this stuff as being test results, why not group it all under a single directory called "results" or "test-results", i.e. so the latest one would be something like results/latest symlinked to results/2015-04-15--1. Also has the benefit of being easier to nuke all your old test results.